src/modules/tracker.js

```diff
@@ -640,18 +642,19 @@ class Tracker {
           sortBy = sort_by,
           selected_filters,
           selectedFilters = selected_filters,
-          url,
+          url = helpers.getWindowLocation()?.href || 'N/A',
```

Comment:
(Out of scope) Should we update browse load + rec view with the same logic as well to keep things consistent?

Reply:
👍 I can make another story.